src/codec_aom.c

```diff
@@ -1025,6 +1025,13 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
     avifBool monochromeRequested = AVIF_FALSE;
 
     if (alpha) {
+        // AV1 specification section 6.4.2. Color config semantics:
+        //   subsampling_x | subsampling_y | mono_chrome | Description
+        //   1             | 1             | 1           | Monochrome 4:0:0
+        //   If matrix_coefficients is equal to MC_IDENTITY, it is a requirement of bitstream
+        //   conformance that subsampling_x is equal to 0 and subsampling_y is equal to 0.
```

Comment:
This comment (lines 1028-1032) explains why we cannot use `AVIF_MATRIX_COEFFICIENTS__IDENTITY`, but it doesn't explain why we should use `AVIF_MATRIX_COEFFICIENTS_UNSPECIFIED`. We should delete this comment. A better reason for using `AVIF_MATRIX_COEFFICIENTS_UNSPECIFIED` is that the alpha image  is not converted to RGB, so `aomImage.mc` is ignored.

Reply:
For the same reason, we should also set `aomImage.cp` and `aomImage.tc` to Unspecified (2) if alpha is true.